### ISSUE TITLE
Export Validation Set

### ISSUE THREAD

**Issue by jameschartouni** [Labelbox/Labelbox#63]
Would you be able to add a feature where you are able to manually choose which images you want to include in your validation set in the review screen? It really helps improve model accuracy that the validation set is more accurately labeled relative to the training set. Thanks!

**Comment by manuaero** [comment]
Hi @jameschartouni, thats a good idea. Perhaps you could add a classification field with name "Validation" and use that to mark if a particular image should be part of the validation set.
